service/dap/server.go

```diff
@@ -1549,13 +1553,80 @@ type stackFrame struct {
 // As per DAP spec, this request only gets triggered as a follow-up
 // to a successful threads request as part of the "request waterfall".
 func (s *Server) onStackTraceRequest(request *dap.StackTraceRequest) {
+	// In order to support paging of stack frames, we always check an additional 10
+	// beyond the number of stack frames that were requested. Since the total frames
+	// will be larger than the number of frames loaded, the client will continue to
```

Comment:
What does "continue to request them" entail, precisely? Does the use have to do something to get more or will the client continue to ask for more until there aren't any more automatically? Because it could be infinite.

Reply:
Ok.